ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -359,6 +359,17 @@
   private static final String KSQL_PROPERTIES_OVERRIDES_DENYLIST_DOC = "Comma-separated list of "
       + "properties that KSQL users cannot override.";
 
+  public static final String KSQL_TOTAL_CACHE_MAX_BYTES_BUFFERING =
+      "ksql.cache.max.bytes.buffering.total";
```

Comment:
I think based on some of the other query configs we have already, these two configs name  would be more suitable
`ksql.query.transient.max.bytes.buffering.total`
`ksql.query.persistent.max.bytes.buffering.total`